### PR TITLE
feat: upgrade `theme-ui` to React 18

### PR DESCRIPTION
**Pull request by khairold**
The current example of Theme UI is on React 17.
It doesn't work with React 18.

I'd like to contribute an example of implementing Theme UI on Remix with React 18.

### CODE REVIEW

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]
Please update the existing example instead of creating a separate React 18 version of it 🙏

**Comment by khairold** [comment]
> Please update the existing example instead of creating a separate React 18 version of it 🙏

Updated

File: theme-ui/README.md [hunk 1]
```diff
@@ -1,50 +1,41 @@
...
+## Preview
 
-## How this implementation works
+Open this example on [CodeSandbox](https://codesandbox.com):
 
-Since [theme-ui](https://theme-ui.com/) is derived from [emotion](https://emotion.sh/), this example shows how we can leverage Emotion's server-side-caching to enable [theme-ui](https://theme-ui.com/) server side rendering.
+<!-- TODO: update this link to the path for your example: -->
 
-_This implementation was based off [Saas-UI Remix guide](https://www.saas-ui.dev/docs/core/installation/remix-guide)._
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/examples/tree/main/theme-ui-react18)
```
Comment (machour) [inline theme-ui/README.md:9]:
```suggestion
[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/examples/tree/main/theme-ui)
```

File: theme-ui/README.md [hunk 2]
```diff
@@ -1,50 +1,41 @@
...
+# Example of Remix (React 18) with Theme UI
 
-This example features how to use [theme-ui](https://theme-ui.com/) with Remix.
+## Preview
 
-## How this implementation works
+Open this example on [CodeSandbox](https://codesandbox.com):
 
-Since [theme-ui](https://theme-ui.com/) is derived from [emotion](https://emotion.sh/), this example shows how we can leverage Emotion's server-side-caching to enable [theme-ui](https://theme-ui.com/) server side rendering.
+<!-- TODO: update this link to the path for your example: -->
```
Comment (machour) [inline theme-ui/README.md:7]:
```suggestion
```

File: theme-ui/README.md [hunk 3]
```diff
@@ -1,50 +1,41 @@
-# Example app with [theme-ui](https://theme-ui.com/)
+# Example of Remix (React 18) with Theme UI
```
Comment (machour) [inline theme-ui/README.md:1]:
```suggestion
# Example with Theme UI
```

**Review by machour — CHANGES_REQUESTED** [review]

**Comment by khairold** [comment]
Fixed lint error and warning

**Comment by khairold** [comment]
Hi this is my first pull request so I'm not sure of the process. 
Just want to check if there is anything else pending on my side.

**Pull request closed without merging**